pkg/logs/sds/scanner.go

```diff
@@ -23,6 +25,21 @@ const ScannedTag = "sds_agent:true"
 
 const SDSEnabled = true
 
+var (
+	tlmSDSConfiguredRules = telemetry.NewCounterWithOpts("sds", "rules_configured", []string{"pipeline"},
+		"Number of configured rules.", telemetry.Options{DefaultMetric: true})
+	tlmSDSMalformedRules = telemetry.NewCounterWithOpts("sds", "rules_malformed", []string{"pipeline"},
+		"Number of malformed rules received through RC.", telemetry.Options{DefaultMetric: true})
+	tlmSDSDisabledRules = telemetry.NewCounterWithOpts("sds", "rules_disabled", []string{"pipeline"},
+		"Rules received but disabled while applying user config.", telemetry.Options{DefaultMetric: true})
+	tlmSDSUnknownStdRule = telemetry.NewCounterWithOpts("sds", "rules_unknown", []string{"pipeline"},
+		"Unknown standard rules while applying user config.", telemetry.Options{DefaultMetric: true})
```

Comment:
Now that these have a `pipeline` tag I'm wondering if a gauge would be a more appropriate data type since I think what customers will be interested in is the number of currently active rules rather than a count of rule applications 🤔 .

Counters for reconfig errors and successes make sense to me.

Reply:
Correct, missed it during the refactor, thank you. It should be addressed + switched to using a `state` on one metric instead.